### DISCUSSION TITLE
HackPrix Hackathon 2024 photo thread! 📸

### DISCUSSION THREAD

**Discussion by toukirkhan** [community/community#127837] [GitHub Education]
### Hi everyone,

Hello HackPrix Hackers!

This thread is for PHOTOS 📸😊. If you're attending our event in Hyderabad today, this is your place to put the best shots of you and your new colleagues/friends. We're gonna pick a random discussion to receive a Mona Figurine!

RULES:
Post anything related to HackPrix in the discussion.
Include your name, GitHub handle, which city you came from and share your experience about HackPrix on the discussion.
One people for the post chosen will receive a mona figurine at the end of the event.

**Comment by SharathChandra0306** [comment]
![20240609_011405.jpg](https://github.com/community/community/assets/75817709/ca8940ad-7b67-4094-bf5f-be39ad48387b)

**Comment by anasfarhan01** [comment]
![IMG_20240608_150024.jpg](https://github.com/community/community/assets/150371872/d8dd2eed-277e-44cf-804c-c4d9d296430b)

![Snapchat-958179704.jpg](https://github.com/community/community/assets/150371872/80f9ed0e-af10-43bf-afa4-70ebc90db9d8)

https://github.com/community/community/assets/150371872/7ef90fc8-bab7-4254-9f7c-30a6fb4d2ef6

![Snapchat-1736796846.jpg](https://github.com/community/community/assets/150371872/0abaf46b-e1dd-4659-af58-c32a61dc4f23)

![Snapchat-244743900.jpg](https://github.com/community/community/assets/150371872/6f909ab0-7a2f-4f5d-a677-173145aa7429)

https://github.com/community/community/assets/150371872/2f798371-77ad-4a70-8e9d-fd78f657acdd

![IMG_20240608_145824.jpg](https://github.com/community/community/assets/150371872/37a603f7-0faa-4565-8f8c-9670f2b43e06)

It's was amazing 
I'm having fun

**Comment by VishalArya123** [comment]
![40de2d9a-8138-49d5-bcf2-57eae91cc551](https://github.com/community/community/assets/137704806/83dca810-eeac-4699-94e8-18ba34eb1f90)

**Comment by Natham-S** [comment]
Sainath S
https://github.com/PixelarRio
I'm from Hyderabad 
Very good organizing by team, very interactive and never felt bored 

Good one guys :)  keep it up 

![IMG_20240608_180605093](https://github.com/community/community/assets/88759952/48849e2f-7315-4540-ab5b-9967de3c8c34)

**Comment by Balu-0512** [comment]
![Uploading IMG_20240608_221247.jpg…]()

**Comment by ArchanaHublikar** [comment]
![Uploading 20240609_020044.jpg…]()

**Comment by FahadNawazKhan** [comment]
![IMG_20240608_221846_619](https://github.com/community/community/assets/84392910/e238a586-34a1-4384-aa4a-7f15fc982d47)

**Comment by FahadNawazKhan** [comment]
![IMG_20240608_222857_299](https://github.com/community/community/assets/84392910/9a2f2889-8677-4599-a29f-6e0a2b71432a)

**Comment by Adithiya5Hub** [comment]
**_I'm S. Adithiya,
From Hyderabad, Here's my GitHub handle_**
[GitHub](https://github.com/Adithiya5Hub)

**_A great session by AdiSuyesh ( HackQuest ) on Web3 and Toukir Khan (GitHub Campus Expert) on GitHub and its uses_**
**_I had the opportunity to interact with them and they were kind enough to stay back and explain all the doubts and give guidance_**
**_It was really informative and I would like to thank Hackprix for conducting the workshop_**

![804598ad-c33c-4ae4-9d6d-11a139805aed](https://github.com/community/community/assets/93365795/5fdc2aa6-7042-4209-b299-e253ca6c1088)

![4d6ddb28-8f80-479d-ba4d-d61869725c64](https://github.com/community/community/assets/93365795/a03da6b3-5b3a-4121-b223-4844d3866b85)


![eb254aa6-0d7d-44ee-8dd5-bc5086a052de](https://github.com/community/community/assets/93365795/7996776a-08cd-418c-91cd-9ad705425de4)

**Comment by P-Potdar** [comment]
Had an amazing conversation with Abdul Wasey and @toukirkhan about how he built a community & managed to conduct a hackathon, gained advice on how to apply for Github Campus expert, which I am surely trying to apply this time in August. 

Three cheers to the entire hackathon team for organizing such a wonderful hackathon. Loved being here. It was Amazing!!

![WhatsApp Image 2024-06-09 at 15 50 27](https://github.com/community/community/assets/109173669/056982a6-d793-4915-b6d9-e6c04797a35c)

**Comment by ebndev** [comment]
Hi @toukirkhan,

Thanks for being a part of the GitHub Community!

While we're glad you were able to use our platform to connect with others at your event, I will be locking this discussion as we want to make sure there is space for users to ask questions without overwhelming them with other conversations.

Please note that this discussion is considered "off-topic" and repeat violations may be considered a violation of our Code of Conduct.

Thanks for understanding!
